src/coreclr/tests/src/GC/Regressions/Github/runtime_32848/runtime_32848.csproj

```diff
@@ -0,0 +1,23 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <CLRTestKind>BuildAndRun</CLRTestKind>
+  </PropertyGroup>
+  <ItemGroup>
+    <Compile Include="$(MSBuildProjectName).cs" />
+  </ItemGroup>
+  
+  <PropertyGroup>
+    <CLRTestBatchPreCommands><![CDATA[
+$(CLRTestBatchPreCommands)
+set COMPlus_gcServer=1
+set COMPlus_gcStress=7
+]]></CLRTestBatchPreCommands>
+    <BashCLRTestPreCommands><![CDATA[
+$(BashCLRTestPreCommands)
+export COMPlus_gcServer=1
+export COMPlus_gcStress=7
```

Comment:
This won't enable any gc stress. The env variable names are case sensitive on Unix and the correct casing is COMPlus_GCStress.

Reply:
Also, I don't think we should hard code GC stress into the test project. The runtests.cmd/sh has an option to run tests with GCStress of any level you want enabled. 
Btw, why GCStress 7 and not F? This means that you want to invoke GC on every allowable JITed instruction, but not on crossgened ones.

Reply:
That for catching that Jan. I think F is too aggressive, but it seemed that 7 was a good level that reproduced the issue. I'm good with changing that to F.
The reason I'm setting gcstress here is because I want a unit test that targets a certain gcstress issue. I can remove that, but at least I need to keep the server GC here.

What's the correct casing for complus_gcserver? is it COMPlus_GCServer?

Reply:
You had the COMPlus_gcServer casing right.
As for F vs 7 levels, why does it seem more aggressive?. The bit 4 just enables / disables GC stress for crossgened code.
We have regular test runs in the lab that are executed with GC stress 3 and C, it sounds like it should be sufficient to catch possible issues.

Reply:
I see, i did not know that bit 4 was to enable/disable gcstress for crossgened code. I will change that to F.

Reply:
This is a meaning of all the bits in the GCStress constant:
https://github.com/dotnet/runtime/blob/39cde70bf08ee8035f5fc3c291903b225c7edccd/src/coreclr/src/vm/eeconfig.h#L390-L397